Project4/styles.css

```
@@ -0,0 +1,151 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+$nav-height: 60px;
+h1 h2 h3 {
+  color: #ffc400;
+}
+body {
+  text-align:center;
+  margin:0px;
+  padding:0px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
```

This is an unnecessarily complicated font stack because it relies _entirely_ on fonts from Google Fonts. Ideally, you'd load one font from Google Fonts, suggest one backup font and then apply the generic font style -- so for Montserrat, you could do something like this:
```suggestion
  font-family: font-family: 'Montserrat', 'montserrat', 'Helvetica Neue', Helvetica, 'Arial Black', Arial, sans-serif;
```
which combines a web font, system fonts and then the default `sans-serif`.